lib/iris/tests/integration/concatenate/test_concatenate.py

```diff
@@ -187,6 +188,53 @@ def test_concatenate_along_anon_dim(self):
         result = concatenate([cube1, cube2])
         self.assertEqual(len(result), 2)
 
+    def _get_3d_cube(self):
```

Comment:
Might be worth also renaming this to be something like `_get_cube_with_anon_dim`